src/main.py

```diff
@@ -151,6 +152,15 @@ async def on_message(message: discord.Message):
             if lvl_up_message:
                 await message.channel.send(lvl_up_message)
 
+        if (message.content.__contains__('https://smapi.io/log/')):
+            message_array = message.content.split()
+            for t in message_array:
+                if (t.__contains__('https://smapi.io/log/')):
+                    log_info = requests.get("http://api.pil.ninja/smapi_log/endpoint?" + t)
+                    json_log_info = log_info.json()
+                    await message.channel.send("SMAPI log info:\nSMAPI version: " + json_log_info['SMAPI_ver'] + "\nStardew Valley version: " + json_log_info['StardewVersion'] + "\nCode mods count: " + json_log_info['SMAPIMods'] + "\nContent Packs: " + json_log_info['ContentPacks'] + "\nOperating System: " + json_log_info['OS'])
```

Comment:
I discussed this with the other moderators yesterday, and while no one is opposed to it, we were concerned that since SMAPI logs get posted *a lot*, we didn't want the chat to be flooded with only slightly useful information. It might be best to condense all of this into only one line, rather than taking up six at a time. For example:

"SMAPI 3.14.7 with Stardew Valley 1.5.6 (build 22018) on Windows, with 116 C# mods and 268 content packs."

Reply:
Thanks for responding, I will change it.

Reply:
Updated it to be as you said, hopefully it's better now.